unit-tests-java8/src/test/java/org/eclipse/collections/test/list/mutable/FixedSizeListTest.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 Goldman Sachs and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.test.list.mutable;
+
+import org.eclipse.collections.api.factory.Lists;
+import org.eclipse.collections.api.list.MutableList;
+import org.eclipse.collections.impl.test.junit.Java8Runner;
+import org.eclipse.collections.test.list.FixedSizeListTestCase;
+import org.junit.runner.RunWith;
+
+@RunWith(Java8Runner.class)
+public class FixedSizeListTest implements FixedSizeListTestCase
+{
+    @SafeVarargs
+    @Override
+    public final <T> MutableList<T> newWith(T... elements)
+    {
+        return Lists.fixedSize.with(elements);
+    }
```

Comment:
By creating this test, I also manage to add a little coverage for the singleton/doubleton/tripleton lists. The assertions are the same ones being applied to ArrayAdapter. The assertions are not written in a way that's careful to use every possible list size, so the coverage from this will be spotty. Still, it's strictly better than not having this test at all.